lib/engines/hcktest/hcktest.rb

```diff
@@ -254,6 +292,10 @@ def run
         dump_run
 
         @project.logger.info("Find all scripts in folder: #{@project.workspace_path}")
+        @project.logger.warn(
+          'Dump mode is only for basic tests. ' \
+          'For specific configurations (e.g., Secure Boot tests), update run scripts manually'
```

Comment:
"run" is superfluous; I don't remember any code that calls the scripts "run scripts". You can just write as "scripts".